vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -165,14 +170,30 @@ private void initExperimentalVariableCard(
     experimentalVariableCard.setAddButtonAction(addVariablesDialog::open);
   }
 
-  private void initExperimentalGroupsBoard() {
-    experimentalGroupsLayoutBoard.setWidthFull();
+  public void experimentId(ExperimentId experimentId) {
+    this.handler.setExperimentId(experimentId);
```

Comment:
groups should be reset somewhere here - the "add card" has to be added again.

Reply:
also: shouldn't this be called setExperimentId(...)? Or am I out of the loop concerning the naming conventions? :)